lib/widgets/chat.dart

```diff
@@ -224,26 +258,60 @@ class _ChatMessageState extends State<ChatMessage> {
                               Theme.of(context).disabledColor.withOpacity(0.5),
                           fontSize: 14,
                         ),
-                      )),
-          if (userSent && isLast)
-            Container(
-              width: double.infinity,
-              margin: const EdgeInsets.only(
-                bottom: 4,
-                left: 18,
-                right: 18,
+                      ),
               ),
-              child: Text(
-                widget.sending ? "Sending..." : "Sent",
-                textAlign: TextAlign.end,
-                style: TextStyle(color: Colors.grey),
+            if (userSent && isLast)
+              Container(
+                width: double.infinity,
+                margin: const EdgeInsets.only(
+                  bottom: 4,
+                  left: 18,
+                  right: 18,
+                ),
+                child: Text(
+                  widget.sending
+                      ? "Sending..."
+                      : "Sent ${timeago.format(msg.createdAt)}",
```

Comment:
Maybe use sentAt better, we're indicating here how long ago it was sent, not when it was created.

Should not affect too much the time but Ithink it might be better

Reply:
Since `timeago` adds ago to the time, so it becomes `Sent 2 minutes ago` it kinda implies that already

Reply:
I mean using `msg.sentAt` instead of `msg.createdAt`

Reply:
Oh Ok